src/App.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './App.css';
+import LeftNav from './Components/Homepage/LeftNav';
+import SubmitBox from './Components/Homepage/SubmitBox';
+import Homepage from './Components/Homepage/Homepage';
+
+function App() {
```

Comment:
I see 2 App files (App.js and App.jsx). 
Are both of them needed? If not - consider removing the redundant one.